terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -742,13 +742,26 @@ def remove_inconsequential_statements(statements: List[JSON]) -> List[JSON]:
                 ]
             }
         },
+        "aws_s3_bucket": {
+            "gitlab": {
+                "bucket": f"edu-ucsc-gi-singlecell-hca-gitlab-{config.deployment_stage}",
+                "acl": "log-delivery-write"
```

Comment:
This is for https://docs.aws.amazon.com/AmazonS3/latest/userguide/enable-server-access-logging.html and not applicable here. The AWS documentation referred to in the issue description has a section on the necessary bucket permissions.